src/aria/utils/json/JsonSerializer.js

```diff
@@ -580,17 +580,41 @@ module.exports = Aria.classDefinition({
             },
 
             /**
-             * Parse a string as JSON. This uses a partial implementation of <a
+             * Parse a string as JSON. This uses a by default the native JSON.parse and when in case of failure uses partial implementation of <a
              * href="https://github.com/douglascrockford/JSON-js/blob/master/json.js">Douglas Crockford's algorithm</a>
              * to parse JSON strings.<br />
              * It provides some security around the eval and resembles what was done in aria.utils.json.load()
              * @param {String} string The string to parse as JSON
              * @return {Object} JSON object
              * @throws SyntaxError
              */
-            parse : function (string) {
+            parse: function (string) {
                 var text = String(string);
+                var JSON = Aria.$global.JSON;
+                if (typeof JSON !== "undefined" && typeof JSON.parse === "function") {
+                    try {
+                        return JSON.parse(text);
+                    } catch (ex) {
+                        // Fallback to eval
+                        this.$logWarn("Cannot parse string with native JSON.parse. Defaulting to eval.\nJSON string:\n%1", [text]);
```

Comment:
Please move this warning string to $statics (just like other error/warning messages).